plugins.txt

```diff
@@ -123,7 +94,6 @@ pipeline-model-extensions:2.2151.ve32c9d209a_3f
 pipeline-rest-api:2.34
 pipeline-stage-step:305.ve96d0205c1c6
 pipeline-stage-tags-metadata:2.2151.ve32c9d209a_3f
-pipeline-stage-view:2.34
```

Comment:
Can we keep this plugin, it provides useful metrics which are easy to get otherwise

Reply:
Surely, but I thought it was superseded by the pipeline-graph-view plugin?

Reply:
Yes, in theory. But this old plugin provides, at a glance, a view of the timing with build history. I'm not aware how to do this with alternatives without  changing pages multiple times.

Reply:
Absolutely, there we go :)